### DISCUSSION TITLE
A few sections in best practices don't quite "define" a best practice...

### DISCUSSION THREAD

**Discussion by asktechsupport** [PoshCode/PowerShellPracticeAndStyle#187] [Up for Debate]
Hello PowerShell folks and @Jaykul 
Love the guidance as a whole, opened my eyes to some better ways of thinking and implementing PowerShell stuffs.

But... I can't help read through and get the feeling some sections of the best practices don't "quite" decide on best practices. they seem a tad confusing and kind of like unfinished thoughts or discussions - am I able to call them out as issues as I see fit, you can always determine "won't fix" for them of course

I'm a keen bean to improve and contribute to this documentation!

**Comment by Jaykul** [comment]
Honestly, it's been awhile since we updated all of this, so it's definitely worth opening discussions.

**Reply by asktechsupport** [reply]
Smashing - how do you best want to handle this, shall we have issues for each page (happy to raise those) or shall we do it through discussions? I'm easy, issues and milestones are pretty beautiful to track the progress though. Ping me back when ya can
